### PR TITLE
Change functions in viewmodels -> Change viewModel method name

### LINKED ISSUE #46: Change functions in viewmodels
Change functions in viewmodels in `MovieDetailsViewModel` and `PopularMoviesViewModel`

Example: 
```
fun getMovieDetails(movieId: Int) = viewModelScope.launch {
        getMovieDetailsUseCase(movieId).fold(
            onSuccess = { movieDetailsModel ->
                _responseMovieDetails.value = movieDetailsModel.toUi()
            },
            onFailure = {
                // TODO changed it when navigator will be implemented
                Log.d("RESPONSE", "Something went wrong")
            }
        )
    }
```
This function returns `Job` and I don't use it in any place therefore it should look like: 
```
fun uploadMovieDetails(movieId: Int) {
        viewModelScope.launch {
            getMovieDetailsUseCase(movieId).fold(
                onSuccess = { movieDetailsModel ->
                    _responseMovieDetails.value = movieDetailsModel.toUi()
                },
                onFailure = {
                    // TODO changed it when navigator will be implemented
                    Log.d("RESPONSE", "Something went wrong")
                }
            )
        }
    }
```
Now it returns` Unit` and when I will use it does not cause mistakes

And it could help: 
Read about lifecycle event observer interface generic class in the view model

- [x] Change viewModel method name
- [ ] Use lifecycle event

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

File: app/src/main/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt [hunk 1]
```diff
@@ -27,24 +27,26 @@ class PopularMoviesViewModel @Inject constructor(
...
-            onSuccess = { moviesModel ->
-                _isLoaderVisible.value = true
-                _popularMovies.value = moviesModel.toUi()
-            },
-            onFailure = {
-                _isLoaderVisible.value = true
-                popularMoviesFragmentNavigator.errorSnackBar("Something went wrong")
-            }
-        )
+    private fun uploadPopularMovie() {
```
Comment (MarekMacko) [inline app/src/main/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt]: "upload" suggests executing some soft of HTTP POST here ;)

Comment (Nataniel-Antosik) [reply app/src/main/java/com/example/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt]: I changed `upload` to `load`

File: app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt [hunk 2]
```diff
@@ -109,8 +105,6 @@ internal class MovieDetailsViewModelTest {
         coEvery { getMovieDetailsUseCase(movieId) } returns Result.success(movieDetailsModel)
```
Comment (MarekMacko) [inline app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt:107]: Names of the tests should be adjusted accordingly :)

Comment (Nataniel-Antosik) [reply app/src/test/java/com/example/benchproject/app/movie/details/viewModel/MovieDetailsViewModelTest.kt:107]: I changed tests names :D

**Review by MarekMacko — APPROVED** [review]

**Pull request merged**